pkg/controller/appdefinition/defaults.go

```diff
@@ -0,0 +1,102 @@
+package appdefinition
```

Comment:
Should this be its own `Defaults` package like it is in the `WorkloadClass` PR or do you prefer it this way?